javascript/linked-list/linked-list-kth/kth.js

```diff
@@ -0,0 +1,44 @@
+class Node {
+  constructor(value, next = null) {
+    this.value = value;
+    this.next = next;
+  }
+}
+
+function kthFromEnd(head, k) {
+  if (!head || k <= 0) {
+    return null;
+  }
+
+  let slowPointer = head;
+  let fastPointer = head;
+
+  for (let i = 0; i < k; i++) {
+    if (!fastPointer) {
+      return null;
+    }
+    fastPointer = fastPointer.next;
+  }
+
+  while (fastPointer) {
+    slowPointer = slowPointer.next;
+    fastPointer = fastPointer.next;
+  }
```

Comment:
We want to stop our loop when our fastPointer is on the last item, not after it's gone off the end. That means we should be checking `fastPointer.next`.